smartcrop.go

```diff
@@ -54,7 +54,7 @@ var (
 
 const (
 	detailWeight            = 0.2
-	skinBias                = 0.01
+	skinBias                = 0.9
```

Comment:
Yes, that's a valid concern. I ended up with this since now the score function does not know whether skin or face was detected. Would it be useful to be able to override these values, or are the same values useful for all use cases? If it is useful some kind of API for providing overrides could solve this problem.

Reply:
Maybe each `Detector` should contain & provide its own bias, which the user could then manipulate, e.g. by manually initializing a `Detector`.